### DISCUSSION TITLE
Desktop version

### DISCUSSION THREAD

**Discussion by hongbo-miao** [open-webui/open-webui#6078] [General]
# Feature Request

## Is your feature request related to a problem? Please describe.
After ChatGPT introduced its desktop version, I found myself using it more frequently.
A desktop version would also open up possibilities for integrations with other apps, like VS Code.
Plus using [Model Context Protocol (MCP)](https://github.com/modelcontextprotocol) for more external data sources and tools.

## Describe the solution you'd like
It would be great to have a desktop version, thanks! ☺️

## Describe alternatives you've considered
Continuing to use the web version.

**Comment by reecelikesramen** [comment]
https://github.com/reecelikesramen/open-webui-desktop. It's brand new but these things are in my roadmap.

**Reply by bhupesh-sf** [reply]
This looks really cool, just starred your repo. Hoping to see more updates to it

**Reply by reecelikesramen** [reply]
Thanks! I plan on continuing this so there is basic desktop support in the mean time. Working on a Windows and Linux pre-release atm. tjbck is also working on an Electron, which I intend to contribute to as well. More on this here: https://github.com/open-webui/open-webui/issues/8262

**Reply by bhupesh-sf** [reply]
Great, thing which I loved the most is its accessibility anywhere in the mac. Keep up the good work. Thanks

**Comment by flefevre** [comment]
Open webui team has already a reflexion on desktop version 
https://github.com/open-webui/open-webui/issues/470

And on MCP
https://github.com/open-webui/open-webui/discussions/7363

**Reply by flefevre** [reply]
There is also this kind of initiatives https://github.com/BrainDriveAI/OpenWebUI_CondaInstaller
And https://github.com/BrainDriveAI/OpenWebUI_DockerInstaller

Hard to monitor all initiatives since open webui is very popular
